FirebaseDatabase/Sources/third_party/SocketRocket/FSRWebSocket.m

```diff
@@ -1559,8 +1559,12 @@ - (void)safeHandleEvent:(NSStreamEvent)eventCode stream:(NSStream *)aStream
 
         case NSStreamEventHasBytesAvailable: {
             SRFastLog(@"NSStreamEventHasBytesAvailable %@", aStream);
+
+            #pragma clang diagnostic push
+            #pragma clang diagnostic ignored "-Wvla"
             const NSUInteger bufferSize = 2048;
```

Comment:
This looks related to building for Objective-C++ instead of Objective C. Do you understand why that's happening?

Either way, it might be cleaner to address with a #define BUFFER_SIZE instead since all usages are in this file.